### PR TITLE
Add requests for upgrade issues

### PR DESCRIPTION
**Pull request by MarcelMue**

### CODE REVIEW

**Review by whites11 — APPROVED** [review]

File: aws/requests.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 releases:
     - name: "> 13.0.0, < 14.0.0"
```
Comment (whites11) [inline aws/requests.yaml:2]: shouldn't it be >= 13.0.0 ?

**Review by tuommaki — APPROVED** [review]
Sure!

File: aws/requests.yaml (hunk 1 above)
Comment (MarcelMue) [reply aws/requests.yaml]: Fixed.

**Review by stone-z — APPROVED** [review]
LGTM

**Pull request merged**